psi-ms.obo

```diff
@@ -22655,6 +22655,386 @@ relationship: has_metric_category MS:4000012 ! single run based metric
+def: "From the distribution of peak densities in MS2, the quartiles Q1, Q2, Q3" [PSI:MS]
+comment: The distribution of peak densities in MS2 can provide insight into the instruments ion selction settings.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_type STATO:0000167 ! first quartile
+relationship: has_type STATO:0000574 ! center value
+relationship: has_type STATO:0000170 ! third quartile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000114
+name: Peak density distribution MS2 mean
+def: "From the distribution of peak densities in MS2, the mean" [PSI:MS]
+comment: The distribution of peak densities in MS2 can provide insight into the instruments ion selction settings.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000115
+name: Peak density distribution MS2 sigma
+def: "From the distribution of peak densities in MS2, the sigma value" [PSI:MS]
+comment: The distribution of peak densities in MS2 can provide insight into the instruments ion selction settings.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000237 ! standard deviation
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000116
+name: Peak density distribution MS2 low outliers (<Q1-1.5*IQR)
+def: "From the distribution of peak densities in MS2, the list of outliers below Q1-1.5*IQR" [PSI:MS]
+comment: The distribution of peak densities in MS2 can provide insight into the instruments ion selction settings.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000036 ! outlier
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+
+[Term]
+id: MS:4000117
+name: Peak density distribution MS2 high outliers (>Q3+1.5*IQR)
+def: "From the distribution of peak densities in MS2, the list of outliers above Q3+1.5*IQR" [PSI:MS]
+comment: The distribution of peak densities in MS2 can provide insight into the instruments ion selction settings.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000036 ! outlier
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+
+[Term]
+id: MS:4000118
+name: Precursor intensity distribution Q1, Q2, Q3
+def: "From the distribution of precursor intensities, the quartiles Q1, Q2, Q3" [PSI:MS]
+comment: The intensity distribution of the precursors informs about the dynamic range of the acquisition.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
```

Comment:
**suggestion:** Add relationship to MS:4000022 (MS2 metric). I assume this is an MS2 metric, because even though the information comes from the MS1 level, without an MS2 scan, there is no notion of a precursor.